modules/storereview/src/main/java/com/gluonhq/attach/storereview/StoreReviewService.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2017, 2019 Gluon
```

Comment:
Please update Copyright header to:

```
Copyright (c) 2021, Gluon
```
for all the files in the StoreReview module.